### DISCUSSION TITLE
Forwarding work for only some channels to other channel/bot

### DISCUSSION THREAD

**Discussion by eliafino** [aahnik/tgcf#340] [Q&A]
Hi,
I don't understand. tgcf not working from some source channels, -100 correct id, but I have tried join link. No way and from console (Verbosity turned on), no msg are captured.
If I forward manually from these channels to my channel/bot, no problem... but no way from tgcf.
I have tried also as "normal user" and with token "bot". If bot, no forward is possible.

I need forwarding only "new messages"

My limitations? Channels limitations? I don't understand!

source "userid" and "-100channel2" working good

`admins: [mytgid] 

forwards:
  - source: userid
    dest: [botusername]
  - source: -100channel1
    dest: [botusername]
  - source: -100channel2
  - dest: [botusername]
  - source: -100channel3
    dest: [botusername]
  - source: -100channel4
    dest: [botusername]
  - source: -100channel5
    dest: [botusername]
  - source: -100channel6
    dest: [botusername]
  - source: -100channel7
    dest: [botusername]

show_forwarded_from: true

live:
  delete_sync: true
`

**Comment by eliafino** [comment]
No forwarding from big channels, with 1k or more users

**Comment by denizkoele** [comment]
@eliafino did you found a solution?

**Reply by eliafino** [reply]
Yes but in another project: edrickrenan/telegram-forwarder

**Reply by denizkoele** [reply]
@eliafino Thanks for the fast answer! But its not based on tgcf.. It dont have these fancy features like here :(

**Reply by eliafino** [reply]
unfortunately, it's not based on tgcf, but at least it works
